src/Components/TodoList/ListItem/ListItem.tsx

```diff
@@ -10,7 +10,7 @@ const removeButtonSx = {
   },
 };
 
-interface ListItemProps {
+export interface ListItemProps {
```

Comment:
This is not well defined, there should be removeButtonClickHandler function that will be passed to the onClick prop of the button.